examples/StandardFirmataBLE/bleConfig.h

```diff
@@ -106,6 +106,15 @@ BLEStream stream;
 // BLEStream stream(BLE_REQ, BLE_RDY, BLE_RST);
 // #endif
 
+/*
+ * Adafruit Bluefruit nRF52 Boards
+ */
+#ifdef ARDUINO_NRF52_ADAFRUIT
+#include "utility/BluefruitStream.h"
+BLEStream stream;
+#endif
```

Comment:
hi @soundanalogous 
If you agree to merge the Boards.h now, it would be super great for us. Then we don't need to include a folk of Firmata. We could point user to our own example now and says we are working on solution to integrate to your stock example. Please merge the Boards.h if possible :+1: